pkg/urbit/vere/pier.c

```diff
@@ -491,6 +491,23 @@ _pier_on_scry_done(void* ptr_v, u3_noun nun)
   else {
     u3l_log("pier: scry succeeded\n");
 
+    //  serialize as desired
+    //
+    u3_atom out;
+    c3_c*   ext_c;
+    {
+      u3l_log("xxx\n");
+      u3_atom puf = u3i_string(u3_Host.ops_u.puf_c);
+      if ( c3y == u3r_sing(c3__jam, puf) ) {
+        out   = u3qe_jam(res);
+        ext_c = "jam";
+      }
```

Comment:
I think technically we leak `puf` in this branch, and maybe `res` overall.  Doesn't really matter for what we're doing, but if we reuse the code later.  At least a comment mentioning it would be good.

Reply:
We're not leaking `res`, because it comes from `nun`, and we `u3z` that at the end of this function. Good catch on `puf` though, I'll just fix that!

Reply:
You can use `u3r_sing_c()` and avoid allocating `puf`.

Reply:
Nevermind, I guess you still need it for `+scot`. You'll need to `u3k` it there.

Reply:
Weird, this wasn't `foul`ing on me without the `u3k`... But I believe you're right.